Sources/AccountSDKIOSWeb/Simplified Login/Model/DataModels.swift

```diff
@@ -31,7 +31,9 @@ extension SimplifiedLoginNamedImageData {
             orderedIconNames = ["Finn", "VG", "Aftenposten", "E24", "BergensTidene"]
         case .proFi:
             orderedIconNames = ["Tori", "Oikotie", "Hintaopas", "Lendo", "Rakentaja"]
-        case .proDk, .pre:
+        case .proDk:
+            orderedIconNames = ["Dba", "Bilbasen"]
```

Comment:
The assets names are "dba" and "bilbasen". With no capital letters.
You should probably change the asset names to have it working

Reply:
Good catch, but only the names of directories matters. Fixed for dba.

Reply:
<img width="120" alt="Screenshot 2022-01-10 at 14 05 36" src="https://user-images.githubusercontent.com/91895636/148770876-305d458d-aaeb-4c24-ac44-405927502fab.png">